ciao-4.11/contrib/lib/python3.5/site-packages/ciao_contrib/cda/data.py

```diff
@@ -199,16 +200,20 @@ def extract_file_type(fname):
     if fn[-1] == "/":
         return None
 
-    if fn == "oif.fits": return "oif"
-    if fn.lower() == "00readme": return "readme"
-    if fn.find("vv") != -1: return "vv"
+    if fn == "oif.fits":
+        return "oif"
+    if fn.lower() == "00readme":
+        return "readme"
+    if fn.find("vv") != -1:
```

Comment:
Unrelated to this PR!

Since we don't care where it's at in the string should we maybe instead use

```python
if 'vv' in fn:
```

instead? (same below in the `known_file_types` loop).

Reply:
Done